src/vscode-dts/vscode.d.ts

```diff
@@ -2056,6 +2079,9 @@ declare module 'vscode' {
 	 *
 	 * @example <caption>A language filter that applies to all package.json paths</caption>
 	 * { language: 'json', pattern: '**​/package.json' }
+	 *
+	 * @example <caption>A language filter that applies to all python files on jupyter notebooks</caption>
+	 * { language: 'python', notebook: 'jupyter-notebook' }
 	 */
 	export interface DocumentFilter {
```

Comment:
I'm a little confused on how `DocumentFilter` plays with `NotebookFilter`. If I set the `NotebookFilter.schema` for example, do I still set `DocumentFilter.shema`? What happens if the two don't match?

I'd sort expect the new type to be `DocumentFilter | NotebookFilter` instead of adding `DocumentFilter.notebook`